src/server/scripts/ScriptLoader.cpp

```diff
@@ -629,6 +629,7 @@ void AddCommandScripts()
     AddSC_reload_commandscript();
     AddSC_reset_commandscript();
     AddSC_spectator_commandscript();
+    AddSC_spy_scripts();
```

Comment:
```suggestion
    AddSC_spy_commandscript();
```

to be on par with other command script names

btw need also to edit the name in the other cpp file

Reply:
@BarbzYHOOL then I'll be including an article on the wiki on how to create a command and what rules to follow

Reply:
And btw, it's not just a command script, it's a player script as well

Reply:
??

I mean that other scripts related to a command (which is what .spy is about, it's a command, if you remove the command it serves no purpose, am I mistaken?) are written like this apparently: `AddSC_commandname_commandscript();`

```
AddSC_reload_commandscript();
AddSC_reset_commandscript();
AddSC_spectator_commandscript();
```

No?

Reply:
Yes, I understand your point. What I meant to explain is why it was written as "*_script*" and not "*_commandscript*"

Reply:
but if it's in two categories, it should be in the "primary" category, which (to me) seems to be "commandscript"


btw when I use the suggest changes button, it's just a suggestion (a quick suggestion without a lot of brain time invested into :P)